src/github/pull-request.ts

```diff
@@ -28,7 +25,7 @@ export default async (context: Context, jiraClient, util): Promise<void> => {
 
 	const jiraPayload = transformPullRequest(
 		context.payload,
-		author.data,
+		context.payload.pull_request?.user,
```

Comment:
We might loose information like avater_url and url for the author

Reply:
nope, it's already part of the `user` object on the payload.  We were doing an extra call for no reason.